### PR TITLE
feat: add assertion for navigation event after click. update various libraries.

### PR DESCRIPTION
**Pull request by admah**
Add ability to handle `assertedEvents` that happen after a user clicks a link. 

Output is: 

```javascript
cy.get("{selector}").click();
cy.location("href").should("eq", "{url}");
```

### CODE REVIEW

File: src/CypressStringifyExtension.ts [hunk 1]
```diff
@@ -89,6 +89,14 @@ export class CypressStringifyExtension extends StringifyExtension {
       );
     }
 
+    if (step.assertedEvents) {
+      step.assertedEvents.map((event) => {
```
Comment (jaffrepaul) [inline src/CypressStringifyExtension.ts:93]: Do you need to iterate with a .map here? if I'm reading it correctly we're just looping over the assertEvents and taking action if we find the type we want. We don't need the new array that the map would create in memory. Maybe a .forEach or a For of loop is a leaner way to go?

**Review by jaffrepaul — CHANGES_REQUESTED** [review]
Just had one question that could be better for performance. Well, not likely to actually matter, but maybe.

File: src/CypressStringifyExtension.ts (hunk 1 above)
Comment (admah) [reply src/CypressStringifyExtension.ts]: Good point, Paul. I switched it out using `forEach`.

**Review by jaffrepaul — APPROVED** [review]
LGTM 🎉

**Pull request merged**